src/i18n/de/README.md

```diff
@@ -0,0 +1,80 @@
+
+Diese Anleitung soll dazu beitragen, dass sich das Ergebnis beim Lesen trotz aller verschiedenen Einflüsse wie ein Gesamtwerk mit gemeinsamem Schreibstil anfühlt.
+
+&nbsp;
+
+
+## Übersetzungs-Glossar
+
+| Originalbegriff             | Übersetzung                          | Anmerkungen
+|:----------------------------|:-------------------------------------|:------------
+| branch                      | der Branch                           | Wird nicht übersetzt (gängiger Domänenbegriff).
+| build                       | Erzeugung / Erstellung               | Wird aus Konsistenzgründen zu bestehenden Übersetzungen vorerst so übersetzt. Eventuell werden wir "Build" aber zukünftig als gängigen Domänenbegriff einstufen und nicht übersetzen.
+| build process               | Erzeugungs- / Erstellungsvorgang     | s.o.
+| build time                  | Erzeugungs- / Erstellungszeitpunkt   | s.o.
+| to build                    | erzeugen                             | Da die Alternativen "bauen" oder gar "builden" merkwürdig klingen, wird diese Übersetzung vermutlich so bleiben.
+| command line                | die Kommandozeile                    |
+| CLI                         | die Kommandozeilen&shy;schnittstelle | Beim ersten Vorkommen in einem Abschnitt kann `(CLI)` dahinter ergänzt werden. Aufgrund der Wortlänge wird die Aufnahme eines weichen Bindestrichs empfohlen: `Kommandozeilen&shy;schnittstelle`
+| CLI flag                    | die Kommandozeilen&shy;option        | Aufgrund der Wortlänge wird die Aufnahme eines weichen Bindestrichs empfohlen: `Kommandozeilen&shy;option`
+| deployment provider         | Hosting-Anbieter                     |
+| to deploy                   | veröffentlichen                      | Wir vermeiden "ausliefern" aufgrund der Zweideutigkeit.
+| domain                      | die Domäne                           |
+| frontmatter                 | das Frontmatter                      | Wird nicht übersetzt (gängiger Domänenbegriff).
+| frontmatter prop(erty)      | die Frontmatter-Eigenschaft          |
+| frontmatter value           | der Frontmatter-Wert                 |
+| page                        | die Seite                            | Eine einzelne (HTML-)Seite. Wir vermeiden die längere Form "Webseite", um Verwechslungen mit "Website" zu vermeiden.
+| repository                  | das Repository                       | Wird nicht übersetzt (gängiger Domänenbegriff).
+| script                      | das Skript                           |
+| site, website               | die Website                          | Eine vollständige Webpräsenz unter einer gemeinsamen Domain, bestehend aus beliebig vielen (HTML-)Seiten.
+| template                    | die Vorlage                          |
+| web                         | das Internet                         |
+
```

Comment:
I would add: UI, Partial Hydration and Isles.

Also configuration info is needed often so we should norm how we name the astro.config.mjs fields. E.g. Parameter, Felder, Schlüssel or Werte...